workspaces/grafana/.changeset/quick-dryers-fix.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-grafana': patch
+---
+
+fixed bug in Constants.ts/alertSelectorFromEntity. This method always returns string before. It can return string or string[] now.
```

Comment:
No, this method does not have a impact because it is already expecting either string or string[].

Reply:
sorry but yes, it has an impact on the `Alerts` component as `alertSelectorFromEntity` is used there. I would suggest to use something more appropriate that describes what the changes do, for example: `Added support for multiple alert selectors`.

Also, perhaps it would be good to mention the new feature in the documentation

Reply:
nice, let's expand these docs here and we are ready to go 🚀 https://github.com/backstage/community-plugins/blob/main/workspaces/grafana/plugins/grafana/docs/alerts-on-component-page.md#with-grafana-unified-alerting-enabled

Reply:
ops sorry, amazing!